extensions/amp-carousel/validator-amp-carousel.protoascii

```diff
@@ -68,3 +68,62 @@ tags: {  # <amp-carousel>
     supported_layouts: RESPONSIVE
   }
 }
+tags: {  # <amp-carousel> with lightbox attribute
+  tag_name: "AMP-CAROUSEL"
+  spec_name: "AMP-CAROUSEL"
+  requires_extension: "amp-carousel"
+  requires_extension: "amp-lightbox-gallery"
+  attrs: {
+    name: "lightbox"
+    mandatory: "true"
+  }
+  attrs: {
+    name: "arrows"
+    value: ""
+  }
+  attrs: {
+    name: "autoplay"
+    value: ""
+  }
+  attrs: { name: "controls" }
+  attrs: {
+    name: "delay"
+    value_regex: "[0-9]+"
+  }
+  attrs: {
+    name: "dots"
+    value: ""
+  }
+  attrs: {
+    name: "loop"
+    value: ""
+  }
+  attrs: {
+    name: "type"
+    value_regex: "slides"
+  }
+  # <amp-bind>
+  attrs: { name: "[slide]" }
+  attr_lists: "extended-amp-global"
+  attr_lists: "lightboxable-elements"
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-carousel"
+  amp_layout: {
+    supported_layouts: FILL
+    supported_layouts: FIXED
+    supported_layouts: FIXED_HEIGHT
+    supported_layouts: FLEX_ITEM
+    supported_layouts: INTRINSIC
+    supported_layouts: NODISPLAY
+    supported_layouts: RESPONSIVE
+  }
+  reference_points {
```

Comment:
Reference point specifying allowed `attr_lists` as "children-of-lightboxed-carousel".